bioio_base/reader.py

```diff
@@ -865,6 +865,45 @@ def physical_pixel_sizes(self) -> PhysicalPixelSizes:
         """
         return PhysicalPixelSizes(None, None, None)
 
+    @property
+    def time_interval(self) -> TimeInterval:
+        """
+        Returns
+        -------
+        sizes: Time Interval
+            Using available metadata, the floats representing time interval sizes for
```

Comment:
not sure I understand this - how many floats are returned?

Reply:
updated in [bb786e6](https://github.com/bioio-devs/bioio-base/pull/20/commits/bb786e6c7bfd147c8f56ad0d463c7d4147a4e2f3)